src/app/routes/stake/stake-list/stake-list.component.scss

```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Rajdhani&display=swap');
```

Comment:
I think this import is not needed.

Reply:
The import is the specific font used in the staking figma design.

Reply:
But we have it in index.html: https://github.com/energywebfoundation/switchboard-dapp/blob/52759d867f5778e191fe4b905fb43863e8043443/src/index.html#L15